api/diagnostics.go

```diff
@@ -67,6 +67,7 @@ type DiagnosticsJob struct {
 	JobEnded              time.Time
 	JobDuration           time.Duration
 	JobProgressPercentage float32
+	fetchVec              prometheus.ObserverVec
```

Comment:
I think the name could be more descriptive and/or type could be more precise. What exactly we will collect there?